json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -45,6 +45,8 @@
 		'jetpack_relatedposts_show_headline' => '(bool) Show headline in related posts?',
 		'jetpack_relatedposts_show_thumbnails' => '(bool) Show thumbnails in related posts?',
 		'jetpack_protect_whitelist'    => '(array) List of IP addresses to whitelist',
+		'jetpack_search_enabled'       => '(bool) Enable Jetpack Search',
+		'jetpack_search_supported'     => '(bool) Jetpack Search is supported',
```

Comment:
Is this supposed to be part of the `request_format` or just the `response_format`?